app/controllers/contact_profiles_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
```

Comment:
Extracted this from the mixed contacts/needs controller, as it is getting confusing.  Makes it easier to rewrite again when we do the full v2 designs

Reply:
So this is the new "update contact and add needs" controller that contact centre agents will use, right?